specification/monitor/resource-manager/Microsoft.Monitor/preview/2021-06-03-preview/operations_API.json

```diff
@@ -0,0 +1,89 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Monitoring Accounts Control Plane API",
+    "description": "Provides operations for working with Azure Monitoring Accounts",
+    "version": "2021-06-03-preview",
+    "x-ms-code-generation-settings": {
+      "name": "MonitorManagementClient"
+    },
+    "contact": {
+      "email": "obsaccounts@microsoft.com"
+    }
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "flow": "implicit",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      },
+      "description": "Azure Active Directory OAuth2 Flow"
+    }
+  },
+  "tags": [
+    {
+      "name": "Operations",
+      "description": "List available operations API"
+    }
+  ],
+  "paths": {
+    "/providers/Microsoft.Monitor/operations": {
```

Comment:
>operations [](http://example.com/codeflow?start=34&length=10)

How are these operations being created? Only list is supported?

Reply:
This is per the RPC (https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/proxy-api-reference.md#exposing-available-operations)